extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -441,7 +441,7 @@ export class AmpAdExit extends AMP.BaseElement {
    * @return {boolean}
    */
   detectAttributionReportingSupport() {
-    return isAttributionReportingAllowed(this.win);
+    return isAttributionReportingAllowed(this.win.document);
```

Comment:
do you happen to know why type checking didnt catch this?

Reply:
no, it looks like the util function is typed correctly with `@param {!Document} doc` 🤷